src/common/style/vanilla/navigation.scss

```diff
@@ -0,0 +1,60 @@
+@import 'vanilla-framework/scss/_utilities_clearfix.scss';
+@import 'vanilla-framework/scss/_base_typography.scss';
+
+// Vanilla 1.5.0 default navigation is dark, so we are using --light navigation,
```

Comment:
BSI is on Vanilla 1.5.0.

I tried to update to latest 1.6.4 in this PR, but it introduced some unrelated issues on home page (bugs reported to vanilla), so I decided to stay on 1.5.0 to reduce the scope of this changes.